bot/bot.py

```diff
@@ -33,6 +33,11 @@
 ⚪ /retry – Regenerate last bot answer
 ⚪ /new – Start new dialog
 ⚪ /mode – Select chat mode
+<<<<<<< HEAD
```

Comment:
Hi!

It seems that there are some unresolved merge conflict markers in the HELP_MESSAGE section of the code.